.github/workflows/continuous-integration.yml

```
@@ -1,6 +1,9 @@
 name: opensim-application
 
-on: [push]
+on:
+  push:
+    branches: 
+      - 'action_build_core'
```

I believe the previous behavior ran on any push (or possibly only to the master branch); possibly a good idea to also have it triggered for pushes to master?
```suggestion
      - 'action_build_core'
      - 'master'
```